src/Stripe.net/Entities/InvoiceItems/InvoiceItemPeriod.cs

```diff
@@ -0,0 +1,13 @@
+namespace Stripe
+{
+    using Newtonsoft.Json;
+
+    public class InvoiceItemPeriod : StripeEntity<InvoiceItemPeriod>
+    {
+        [JsonProperty("end")]
+        public long End { get; set; }
+
+        [JsonProperty("start")]
+        public long Start { get; set; }
```

Comment:
How should these now be converted to date times? Is there a helper function for this that you have built in? This really makes it so it's not very idiomatic